api/server.py

```diff
@@ -63,9 +92,18 @@ def get_api_token():
     return response.json().get("access_token", None)
 
 
-CORS_URL = "*"
-ALGORITHMS = ["RS256"]
-APP = Flask(__name__)
+def notAuthorizedError():
+    """Return 403 error as application/json in shape of the Auth0 API"""
+    return (
+        jsonify(
+            {
+                "error": "Forbidden",
+                "message": "You do not have permission to access this resource.",
+                "statusCode": 403,
+            }
+        ),
+        403,
+    )
```

Comment:
before this change, our API was not returning JSON on `403` errors. doing so makes it easier to write the frontend response handler.